sunpy/sun/models.py

```diff
@@ -198,3 +200,68 @@ def differential_rotation(duration: u.s, latitude: u.deg, *, model='howard', fra
         rotation -= 0.9856 * u.deg / u.day * duration
 
     return Longitude(rotation.to(u.deg))
+
+# This code block adds a 1D structure of the chromosphere
+# using the Avrett & Loeser (2008) paper.
+
+CHROMO_DATA_DIR = pathlib.Path(__file__).parent.absolute() / "data"
+csv_file_path = CHROMO_DATA_DIR / "chromosphere_avrett_Loeser_2008_model.csv"
+
+
+# Define units for the dataset columns
+_units = {
+    "h": u.km,
+    "m": u.g / u.cm**2,
+    "T": u.K,
+    "V": u.km / u.s,
+    "p_g": u.dyne / u.cm**2,
+    "p_tot": u.dyne / u.cm**2,
+    "n_H": u.cm**-3,
+    "n_HI": u.cm**-3,
+    "n_e": u.cm**-3,
+}
+
+# Function to read the chromosphere model CSV file
+def _read_chromosphere_data(file=csv_file_path):
```

Comment:
Can you read this file, add units and then save the QTable out?
That might add the units to the file and then we won't need to define the units for each column